src-ui/app/views/dashboard-view/views/about-view/about-view.component.scss

```
@@ -116,195 +60,225 @@
+    flex: 2;
+  }
+
+  &-spacer-bottom {
+    flex: 3;
+  }
 
   &-avatar {
     @include shadows.shadow(4, true);
-    overflow: hidden;
-    border-radius: 999999999px;
+    margin-bottom: 0.9em;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)